artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -336,6 +336,14 @@
             </xsd:annotation>
          </xsd:element>
 
+         <xsd:element name="address-queue-scan-period" type="xsd:long" default="30000" maxOccurs="1" minOccurs="0">
```

Comment:
Can this be added to the artemis-configuration used for testing to keep as much as possible the two in sync